web/src/app/shared/components/tree/tree.component.ts

```diff
@@ -111,7 +109,7 @@ export class TaskanaTreeComponent implements OnInit, AfterViewChecked, OnDestroy
   classificationsDeepCopy(classification: TreeNodeModel) {
     const ret: TreeNodeModel = { ...classification };
     ret.children = ret.children ? [...ret.children] : [];
-    ret.children = ret.children.map(this.classificationsDeepCopy.bind(this));
+    ret.children = ret.children.map(children => this.classificationsDeepCopy(children));
```

Comment:
This is a little confusing. We are mapping every single child, not all :) But the naming is fine for now.